packages/blockchain-utils-validation/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import { LinkProof } from "@ceramicnetwork/blockchain-utils-linking";
+import ethereum from "./blockchains/ethereum";
+import filecoin from "./blockchains/filecoin";
+import polkadot from "./blockchains/polkadot";
+import eosio from "./blockchains/eosio";
+import { AccountID } from "caip";
+
+const handlers = {
+  [ethereum.namespace]: ethereum,
+  [filecoin.namespace]: filecoin,
+  [polkadot.namespace]: polkadot,
+  [eosio.namespace]: eosio,
+};
+
+const findDID = (did: string): string | undefined =>
+  did.match(/(did:(3|muport):[a-zA-Z0-9])\w+/)?.[0];
```

Comment:
Would you mind changing this as per: https://github.com/ceramicnetwork/js-3id-blockchain-utils/pull/45 🙏

Reply:
Done.